client/state/plugins/installed/actions.js

```diff
@@ -51,6 +51,25 @@ import { getSite } from 'calypso/state/sites/selectors';
 
 import 'calypso/state/plugins/init';
 
+/**
+ * Determines the truthiness of a site specific property regardless of whether it is on the plugin object
+ * or on one of the plugin's site objects.
+ *
+ * @param {string} prop - The site property to check. One of 'active', 'autoupdate', 'update', or 'version'.
+ * @param {Object} plugin - The plugin object
+ * @param {number} siteId - The ID of the site
+ * @returns {any} The value of the prop being requested
+ */
+const pluginHasTruthySiteProp = ( prop, plugin, siteId ) => {
```

Comment:
I appreciate addressing the documentation issues, but I think the function name still suffers from the same issue as before. The function name hints to expect a boolean, but it actually returns the prop (if there). Also, the function description still doesn't properly address that problem IMHO - it doesn't make it clear as to what the function returns. Furthermore, if I understand correctly, we seem to be using the function as a boolean (below in this PR), but it will return the prop value which may be of any type, so we rely on implicit type casting to boolean - is there a good reason for that? I think that area of the PR still needs some work.

Reply:
In this case I intended the function to return a boolean, so I have updated it to actually do that.